ipc/provider/src/checkpoint.rs

```diff
@@ -203,18 +199,22 @@ impl<T: BottomUpCheckpointRelayer + Send + Sync + 'static> BottomUpCheckpointMan
                     .await?;
                 log::debug!("bottom up bundle: {bundle:?}");
 
-                todo!("implement submiet checkpoint")
```

Comment:
Was this on `main`? 😨

Reply:
I think this was in between the BottomUpBatch and BottomUpCheckpoint refactoring, and the contract merged but relayer not ready yet, so it's unimplemented first.